src/test/fixture/addressbook/Maersk Bill of Lading.tt

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Should this file be inside this folder? We may not have to put the .csv file in /addressbook as well. Can move them one level up.